src/Client.php

```diff
@@ -202,14 +202,21 @@ private function receive(): stdClass|array|int
         $result = BSON\toPHP(substr($res, 21, $responseLength - 21));
 
         if (property_exists($result, "writeErrors")) {
-            // Throws a Utopia\Mongo\Exception with Code Error
-
+            // Throws Utopia\Mongo\Exception
             throw new Exception(
                 $result->writeErrors[0]->errmsg,
                 $result->writeErrors[0]->code
             );
         }
 
+        if (property_exists($result, 'errmsg')) {
```

Comment:
What's the difference between `writeErrors` and `errmsg`?

Reply:
They are represented in 2 different structures.
one is an array and the second is not for example.

Reply:
insde the array there is an entry for "errmsg"